adapters/beintoo/beintoo.go

```
@@ -0,0 +1,254 @@
+package beintoo
+
+import (
+	"encoding/json"
+	"fmt"
+	"net/http"
+	"net/url"
+	"strconv"
+	"time"
+
+	"github.com/mxmCherry/openrtb"
+	"github.com/prebid/prebid-server/adapters"
+	"github.com/prebid/prebid-server/errortypes"
+	"github.com/prebid/prebid-server/openrtb_ext"
+)
+
+type BeintooAdapter struct {
+	endpoint string
+	testing  bool
+}
+
+func buildEndpoint(endpoint string, testing bool, timeout int64) string {
+
+	if timeout == 0 {
+		timeout = 1000
+	}
+
+	uriObj, _ := url.Parse(endpoint)
```

Please do not ignore the error response.